home_audit.node.js

```diff
@@ -1605,8 +1627,11 @@ function validate_home_audit (homeValues, additionalRules = null) {
  */
 function validate_address (homeValues) {
     let mandatoryMessage = "Missing value for mandatory field";
+    let optionalMessage = "Required field, if no cooridnates are provided."
```

Comment:
Incorrect spelling of "coordinates."  The grammar here is also weird, so I would recommended: "Field is required if no coordinates are provided."